modules/markup/markdown/markdown_test.go

```diff
@@ -21,12 +21,11 @@ import (
 )
 
 const (
-	AppURL    = "http://localhost:3000/"
-	Repo      = "gogits/gogs"
-	AppSubURL = AppURL + Repo + "/"
+	AppURL  = "http://localhost:3000/"
+	FullURL = AppURL + "gogits/gogs/"
```

Comment:
What does this mean?

Why "AppURL" is "http://localhost:3000/" but there is "gogits/gogs/" in FullURL?

* If the "gogits/gogs/" is for sub URL, then it should define AppSubURL to "/gogits/gogs"
* If the "gogits/gogs/" is for a repo like "user/repo", then it shouldn't be provided here.

Reply:
Thank you. Maybe it could be called `RepoFullURL`. I have no more question now.